### ISSUE TITLE
proxyfmu seems to not retry new ports

### ISSUE THREAD

**Issue by ljamt** [open-simulation-platform/proxy-fmu#50]
After https://github.com/open-simulation-platform/proxy-fmu/pull/48 got merged I've never seen proxyfmu retry binding to ports. Something that happened all the time previously. 

Prior to https://github.com/open-simulation-platform/proxy-fmu/pull/48 the errors related to binding to ports resulted in retries:

```
[proxyfmu] Booting FMU instance 'vessel_model'..
[proxyfmu] Could not bind: errno = 10013. Failed to bind to port 64724. Retrying with another one. Attempt 1 of 10..
[proxyfmu] FMU instance 'vessel_model' instantiated using port 57651
```

After https://github.com/open-simulation-platform/proxy-fmu/pull/48 the process returns with status code 2:

```
[proxyfmu] Booting FMU instance 'vessel_model'..
[proxyfmu] External proxy process for instance 'vessel_model' returned with status 2
```

**Comment by markaren** [comment]
Ahhhh. Serve() throws the error..... Which is now outside..

**Comment by markaren** [comment]
Fixed in 87595af
